### PR TITLE
TrilioVault for Kubernetes installation script improved

### PR DESCRIPTION
**Pull request by umeshkumhar**
## BACKGROUND
- Remove hardcoded dependency of Chart Version

-----------------------------------------------------------------------

## Changes
- Removed the hardcoded version and pull the latest version on runtime
- updated same with upgrading script

-----------------------------------------------------------------------

## Checklist
[X] deploy.sh along with configured management console/TVK UI and preinstalled TVK license.
[X] upgrade.sh

------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

File: stacks/triliovault-operator/deploy.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ helm repo update > /dev/null
 ################################################################################
 STACK="triliovault-operator"
 CHART="triliovault-operator/k8s-triliovault-operator"
-CHART_VERSION="2.6.7"
+CHART_VERSION="$(curl -s https://charts.k8strilio.net/trilio-stable/k8s-triliovault-operator/index.yaml | grep -m 1 appVersion | awk -F ':' '{gsub(/ /,""); print $2 }')"
```
Comment (keladhruv) [inline stacks/triliovault-operator/deploy.sh:16]: Removing the chart version altogether should install the latest version.

**Review by keladhruv — APPROVED** [review]

**Review by keladhruv — COMMENTED** [review]
Approved it by mistake, waiting on the latest tag comment

**Review by keladhruv — CHANGES_REQUESTED** [review]
Approved it by mistake, waiting on the latest tag comment. (Let's hope this time it blocks merges)  :D

File: stacks/triliovault-operator/deploy.sh (hunk 1 above)
Comment (umeshkumhar) [reply stacks/triliovault-operator/deploy.sh]: yes actually, we need a version only for an upgrade, added `latest`

Can you help me understand how the upgrade works on DO? how DO will come to know that an application upgrade is available?

Comment (umeshkumhar) [reply stacks/triliovault-operator/deploy.sh]: @keladhruv resolved issues

**Review by keladhruv — APPROVED** [review]

**Pull request merged**